rollup.config.js

```diff
@@ -0,0 +1,18 @@
+export default {
+  input: 'index.js',
+  external: ['fluture/index.js', 'callgebra/index.js'],
+  output: {
+    format: 'umd',
+    name: 'flutureHooks',
+    file: 'index.cjs',
+    interop: false,
+    paths: {
+      'fluture/index.js': 'fluture',
+      'callgebra/index.js': 'callgebra',
```

Comment:
This is what I meant with choice iii in https://github.com/fluture-js/Fluture/pull/383#issue-324290164